packages/core/src/Radio/__snapshots__/Radio.spec.tsx.snap

```diff
@@ -198,7 +198,7 @@ exports[`Radio Selected, rendering 1`] = `
 }
 
 .c0:hover > svg {
-  color: #049 !important;
+  color: #0068ef !important;
```

Comment:
Ooh, is this an actual change caught by a snapshot?

Reply:
I think it might be! I wasn't paying close attention because we're going to address Chromatic diffs on the `DSv6` branch, but I can address this before merging.